src/gui/FileBrowser.cpp

```diff
@@ -403,11 +340,11 @@ void FileBrowserTreeWidget::contextMenuEvent(QContextMenuEvent * e )
 						this,
 					SLOT( sendToActiveInstrumentTrack() ) );
 		contextMenu.addAction( tr( "Open in new instrument-track/"
-								"Song-Editor" ),
+								"Song Editor" ),
 						this,
 					SLOT( openInNewInstrumentTrackSE() ) );
 		contextMenu.addAction( tr( "Open in new instrument-track/"
-								"B+B Editor" ),
+								"Pattern Editor" ),
```

Comment:
@Wallacoloo I'm not sure this got caught, but this cause inconsistencies in the interface... 

![image](https://cloud.githubusercontent.com/assets/6345473/10140878/af8dfbca-65d7-11e5-9ef5-6888862cfe86.png)